data/mods/TEST_DATA/items.json

```
@@ -516,11 +557,12 @@
     "description": "A poorly balanced sword for test purposes",
     "category": "spare_parts",
     "weight": "1250 g",
+    "volume": "1 L",
+    "longest_side": "100 cm",
```

```suggestion
    "longest_side": "1 meter",
```
i remember what you said about tests doubling as documentation. i also like having the largest denomination of unit possible.